src/compute/src/compute_state.rs

```diff
@@ -239,6 +245,29 @@ impl<'a, A: Allocate + 'static> ActiveComputeState<'a, A> {
             }
             None => (),
         }
+        match enable_columnation_lgalloc {
+            Some(true) => {
+                if let Some(path) = self.compute_state.context.scratch_directory.as_ref() {
+                    info!(path = ?path, "Enabling lgalloc");
+                    lgalloc::lgalloc_set_config(
+                        lgalloc::LgAlloc::new()
+                            .enable()
+                            .with_path(path.clone())
```

Comment:
we probably want this to be `path.join("compute")`, and I can fix the equivalent for storage!

Reply:
https://github.com/MaterializeInc/materialize/pull/23117

Reply:
Sure, we can change this, but lgalloc only creates unlinked files, so it shouldn't matter too much.